.travis.yml

```diff
@@ -22,6 +23,10 @@ matrix:
 script:
   - bash ./format.sh
 
+  # Check for possible typos
+  - pip install codespell
+  - codespell -I codespell-ignore.txt {about,community,development,getting_started,tutorials}/**/*.rst
```

Comment:
Does `**/*.rst` also work recursively deeper than 2 levels? (e.g. `tutorials/vr/vr_starter_tutorial/*.rst`)

Reply:
Just checked, it should :)

Reply:
Ideally, I'd have have used `^classes/**/*.rst`, but it seems to be a zsh-ism :slightly_frowning_face:

Reply:
You may use `--skip "classes/*.rst"`.

Reply:
Or `--skip ".git,classes"` seems to work too and also ignored `.git` (locally I'd add `_build`, but on Travis we shouldn't need it).

Reply:
That doesn't seem to work, let's keep it as it currently is then.